webextensions/api/scripting.json

```
@@ -119,6 +119,25 @@
               "safari_ios": "mirror"
             }
           },
+          "cssOrigin": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "144"
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": false
```

```suggestion
                  "version_added": "18"
```

Let's do this. Can change later if inaccurate (which would be surprising considering that the WebKit change was all in the same commit: https://searchfox.org/wubkat/commit/f4d3ac245e7d24d2d7702b884c10f29527be307a )